activemodel/lib/active_model/dirty.rb

```diff
@@ -13,7 +13,7 @@ module ActiveModel
   # * <tt>include ActiveModel::Dirty</tt> in your object.
   # * Call <tt>define_attribute_methods</tt> passing each method you want to
   #   track.
-  # * Call <tt>[attr_name]_will_change!</tt> before each change to the tracked
+  # * Call <tt>[attribute_name]_will_change!</tt> before each change to the tracked
```

Comment:
What happens if we remove the `<tt>` here? Can we link to the methods since they are documented now?

Reply:
Sadly that doesn't work:
<img width="583" alt="image" src="https://github.com/rails/rails/assets/28561/9e70a8ae-a6bd-4eae-a79d-3c77ed8c9be5">

Reply:
It's a bit confusing but I was able to find a way to link to the HTML id for that method using:

```
{[attribute_name]_will_change!}[rdoc-label:method-i-5Battribute_name-5D_will_change-21]
```

I'm not thrilled with it though, and I think it's kind of relying too heavily on rdoc internals to generate that id. 🤔

Reply:
Thanks! That works great.
I've decided to not link it though, because that we don't link to the other methods in that list either.
It's good to know we can use it in other places, though.

Reply:
Applied this now to some of the dispatch target methods:
<img width="517" alt="image" src="https://github.com/rails/rails/assets/28561/445485bb-53bf-43ac-9d3f-71ace3565d6f">